.circleci/config.yml

```diff
@@ -357,7 +357,7 @@ workflows:
           name: Deploy backend production
           gitsha: $CIRCLE_SHA1
           stack_name: "tm4-production"
-          host_ami: "/aws/service/canonical/ubuntu/server/20.04/stable/current/amd64/hvm/ebs-gp2/ami-id"
+          host_ami: "/aws/service/debian/release/11/20240813-1838/amd64"
```

Comment:
Why not `/aws/service/debian/release/11/latest/amd64` (currently set to the 2024-10-07 Debian image)?

Reply:
My logic is that using latest could introduce a breaking or unintended change. Even at a low chance I would rather avoid it, and this code is going to be depreciated soon anyways so no long-term planning needed.